src/elm/App.elm

```diff
@@ -418,6 +430,52 @@ view address model =
         ]
 
 
+    projects = span [] (List.map projectsButtons model.projects)
+
+
+    padButtons =
+      let
+        digitButton digit =
+          let
```

Comment:
@amitaibu 
I think It won't work if they are on the same level.
`padButtons` is a wrapper declaration that has inner declarations that helps to generate the output of  `padButtons`. Can you show me how to do it ?